ffxivbot/consumers.py

```diff
@@ -287,10 +287,12 @@ async def receive(self, text_data):
                 if echo.find("get_group_list") == 0:
                     self.bot.group_list = json.dumps(receive["data"])
                     self.bot.save(update_fields=["group_list"])
-                if echo.find("_get_friend_list") == 0:
+                if echo.find("get_friend_list") == 0:

```

Comment:
So the API name of mirai is `get_friend_list`? I don't want to break the current APIs since CoolQ is not fully broken.

Reply:
CQHTTP >= 4.14.1 can support,the response is a JSON array.